content/docs/start/experiments/experiment-versioning.md

```diff
@@ -0,0 +1,224 @@
+---
+title: 'Get Started: Experiment Versioning'
+description:
+  'Get started with versioning experiments with DVC. Track the changes to the
+  code, data, metrics, parameters and plots associated with each experiment,
+  without bloating your Git repo.'
+---
+
+# Get Started: Experiment Versioning
+
+The early stages of a Machine Learning (ML) project are usually exploratory and
+demand fast iterations. Many ML practitioners chose to use tools that require
+little setup and enable rapid prototyping, like those based on
+[Jupyter Notebooks](https://jupyter.org/).
+
+Although these tools are extremely useful, keeping track of the changes you made
+across experiments quickly becomes cumbersome and reproducing an experiment can
+be a challenge.
+
+You can start using DVC to version your experiments without leaving your Jupyter
+Notebook. There are no logins, servers, databases, or UI to spin up. Every
+experiment will be versioned **without cluttering your repo**, unlike saving
+each run to a separate directory or creating a Git branch for each.
+
+## Enable Versioning
+
+All you need to start is a <abbr>DVC repository</abbr> and the [DVCLive] Python
+library installed:
+
+```cli
+$ pip install dvclive
+```
+
+In your Python code, you can start versioning your experiments with
+`save_dvc_exp=True`.
+
+There are some examples below
+([other frameworks available](/doc/dvclive/api-reference/ml-frameworks)):
+
+<toggle>
+
+<tab title="Pytorch Lightning">
+
+```python
+from dvclive.lightning import DVCLiveLogger
+
+...
+
+trainer = Trainer(logger=DVCLiveLogger(save_dvc_exp=True))
+trainer.fit(model)
+```
+
+</tab>
+
+<tab title="Hugging Face">
+
+```python
+from dvclive.huggingface import DVCLiveCallback
+
+...
+
+trainer.add_callback(DVCLiveCallback(save_dvc_exp=True))
+trainer.train()
+```
+
+</tab>
+
+<tab title="Keras">
+
+```python
+from dvclive.keras import DVCLiveCallback
+
+...
+
+model.fit(
+  train_dataset, validation_data=validation_dataset,
+  callbacks=[DVCLiveCallback(save_dvc_exp=True)])
+```
+
+</tab>
+
+<tab title="General Python API">
+
+```python
+from dvclive import Live
+
+with Live(save_dvc_exp=True) as live:
+    live.log_param("epochs", NUM_EPOCHS)
+
+    for epoch in range(NUM_EPOCHS):
+        train_model(...)
+        metrics = evaluate_model(...)
+        for metric_name, value in metrics.items():
+            live.log_metric(metric_name, value)
+        live.next_step()
+```
+
+</tab>
+
+</toggle>
+
+After this, each execution of the code will create a DVC <abbr>experiment</abbr>
+containing the results and the changes needed to reproduce it.
```

Comment:
Related to the [comment above](https://github.com/iterative/dvc.org/pull/4393/files#r1139207329), this would be a good alternative place to introduce that experiments are saved with git.